atd-vze/src/views/Crashes/Crash.js

```diff
@@ -254,14 +254,14 @@ function Crash(props) {
                 </Col>
               </Row>
             </CardHeader>
-            <CardBody style={{ minHeight: "350px" }}>
-              {(!latitude || !longitude) && (
+            <CardBody className="d-flex flex-column">
```

Comment:
This fixes a bug where the map was escaping its parent card when the "no coordinates" alert showed. There is a screenshot in the issue. Now, the map will stretch to fit the remaining height in the card.

Reply:
disregard - starting a new comment since we need the `minHeight` afterall 🙃